index.html

```diff
@@ -32,9 +37,27 @@
         <div>
         <img src="hackathon_icon.jpg" alt="A man jumping over a hurdle with a label of hackathon">
```

Comment:
This img element should be indented. I learned that things within the div element are children to the div.